application/shoot.c

```diff
@@ -459,39 +451,53 @@ static void trigger_motor_turn_back(void)
 	}
 }
 
-
-static void shoot_bullet_control(void)
+static void load_bullet_control(void)
 {
     // Rotate by TRIGGER_ANGLE_INCREMENT every time
-    if (shoot_control.move_flag == 0)
-    {
-        shoot_control.set_angle = rad_format(shoot_control.angle + TRIGGER_ANGLE_INCREMENT);
-        shoot_control.move_flag = 1;
-    }
+    
 
-    if ((shoot_control.press_l == 0) && (shoot_control.shoot_rc->rc.s[RC_LEFT_LEVER_CHANNEL] != RC_SW_UP) && (shoot_control.key == SWITCH_TRIGGER_OFF))
+    if ((shoot_control.press_r == 0) && (shoot_control.shoot_rc->rc.s[RC_LEFT_LEVER_CHANNEL] != RC_SW_UP) && (shoot_control.key == SWITCH_TRIGGER_OFF))
     {
         shoot_control.shoot_mode = SHOOT_STOP;
     }
+    if((shoot_control.press_r == 1)&&(shoot_control.press_l == 0))
```

Comment:
Use `else if`, otherwise, this if statement will overwrite the behaviour of `(shoot_control.shoot_rc->rc.s[RC_LEFT_LEVER_CHANNEL] != RC_SW_UP) && (shoot_control.key == SWITCH_TRIGGER_OFF)`